webrtc.html

```diff
@@ -8793,57 +8793,49 @@ <h3><dfn>RTCIceTransport</dfn> Interface</h3>
         </li>
         <li>
           <p>Let <var>newState</var> be the new indicated
-          <code><a>RTCIceTransportState</a></code>.
+          <code><a>RTCIceTransportState</a></code>.</p>
         </li>
         <li>
-          <p>Set <var>transport</var>.<a>[[\IceTransportState]]</a> to
-          <var>newState</var>.</p>
-        </li>
-        <li>
-          <p><a>Fire an event</a> named <code><a>statechange</a></code> at
-          <var>transport</var>.</p>
+          <p>If <var>newState</var> is <code>"new"</code>,
+          <code>"checking"</code> or <code>"closed"</code>,
+          set <var>transport.<a>[[\SelectedCandidatePair]]</a></var>
+          to <code>null</code>.</p>
         </li>
         <li>
-          <p><a>Update the ICE connection state</a> of <var>connection</var>.
-          </p>
+          <p>If <var>newState</var> is <code>"connected"</code>, set
+          <var>transport.<a>[[\SelectedCandidatePair]]</a></var> to the
+          best valid pair (prior to nomination) or the nominated candidate
+          pair.</p>
         </li>
         <li>
-          <p><a>Update the connection state</a> of <var>connection</var>.</p>
+          <p>If <var>newState</var> is <code>"completed"</code>,
+          <code>"disconnected"</code> or <code>"failed"</code>,
+          <var>transport.<a>[[\SelectedCandidatePair]]</a></var> retains
+          its last set value.</p>
```

Comment:
This implies that we must go to "connected" before "completed", because only in "connected" do we update the candidate pair. Is this true? What if there is only a single pair to check, we check it, and it connects. There's no pairs to check, so we're completed. Would connected -> completed necessarily be two steps and not one?

Reply:
Also saying "retains its last value" is a NO-OP, see suggestion for clarifying what the pair's value is as a note instead.